src/main/java/uk/gov/hmcts/reform/pip/data/management/helpers/JsonParser.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.pip.data.management.helpers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class JsonParser {
+
+    private JsonParser() {
+
+    }
+
+    public static String readAttribute(String json, String attribute)
```

Comment:
If possible I think this should be the user model rather than parsing the JSON manually